pkgs/applications/audio/csound/csound-qt/rtmidipath.patch

```diff
@@ -3,15 +3,15 @@ index e5e0c896..9a9fa513 100644
 --- a/src/src.pri
 +++ b/src/src.pri
 @@ -155,9 +155,9 @@ pythonqt {
-         "src/pyqcsobject.cpp"
```

Comment:
Why do you convert these lines to use CRLF?

Reply:
I got a weird error. That was only solved by changing the newline chars. Maybe it was because two types of newline chars weren't accepted in some part of the build process.

Reply:
It would be helpful to see that error. It is strange that using CRLF fixes anything given that UNIX uses LF.

Reply:
The error

```
applying patch /nix/store/xnmk6awjms139qlwrl70jnm4067gymmy-rtmidipath.patch
patching file src/src.pri
Hunk #1 FAILED at 155 (different line endings).
1 out of 1 hunk FAILED -- saving rejects to file src/src.pri.rej
```

Reply:
I guess this is okay then!